src/lib/routes.ts

```diff
@@ -1,5 +1,7 @@
 export default {
 	HOME: '/',
+	LOGIN: '/login',
+	REGISTRATION: '/signup',
```

Comment:
Generally use either `register` & `login` or `sign up` & `sign in`

```suggestion
	LOGIN: '/login',
	REGISTER: '/register',
```